Extension/src/LanguageServer/client.ts

```diff
@@ -1109,6 +1109,7 @@ export class DefaultClient implements Client {
         const settings_intelliSenseEngine: (string | undefined)[] = [];
         const settings_intelliSenseEngineFallback: (string | undefined)[] = [];
         const settings_errorSquiggles: (string | undefined)[] = [];
+        const settings_headerSourcePaneJumping: (boolean | undefined)[] = [];
```

Comment:
All the edits to client.ts are not needed and should be removed. Those client.ts settings are just for settings which need to be sent to our cpptools process.